packages/core/src/canvas/index.ts

```diff
@@ -508,6 +509,14 @@ export default class CanvasModule extends Module<CanvasConfig> {
     };
   }
 
+  startSort(dragSource: DragSource<Component>) {
+    this.em.set('dragSource', dragSource);
+  }
+
+  endSort() {
+    this.em.set({ dragResult: null, dragSource: undefined });
+  }
```

Comment:
`startSort` > `startDrag`
`endSort` > `endDrag`

let's also put a small jsdoc explaining these methods could be used for custom D&D content